torch/csrc/distributed/rpc/rref_impl.cpp

```diff
@@ -225,6 +234,18 @@ void OwnerRRef::setValue(IValue&& value) {
   }
 }
 
+void OwnerRRef::setError(std::string error) {
+  std::unique_lock<std::mutex> lock(mutex_);
+  error_ = error;
```

Comment:
std::move here to avoid copy?

Reply:
Can't unfortunately due to the use in the callback below.
I can const ref, can update